[2019-06-03 10:50:18] ERROR - scripture.pipelines.mongo_pipeline: Exception
Traceback (most recent call last):
  File "/Users/huangjack/PycharmProjects/scripture/scripture/pipelines/mongo_pipeline.py", line 205, in process_item
    yield self._processor(item)
IndexError: list index out of range
[2019-06-03 10:50:19] ERROR - scripture.pipelines.mongo_pipeline: Exception
Traceback (most recent call last):
  File "/Users/huangjack/PycharmProjects/scripture/scripture/pipelines/mongo_pipeline.py", line 205, in process_item
    yield self._processor(item)
IndexError: list index out of range
